Content.Server/Punpun/PunpunComponent.cs

```diff
@@ -0,0 +1,4 @@
+namespace Content.Server.Punpun;
+
+[RegisterComponent]
+public sealed partial class PunpunComponent : Component;
```

Comment:
I had no idea you could just close a component with a semicolon like that, I thought you had to close them with { }
Huh.